web/src/app/shared/store/classification-store/classification.state.ts

```diff
@@ -45,16 +59,15 @@ export class ClassificationState implements NgxsAfterBootstrap {
   }
 
   @Action(SelectClassification)
-  selectClassification(ctx: StateContext<ClassificationStateModel>, action: SelectClassification): Observable<any|null> {
+  selectClassification(ctx: StateContext<ClassificationStateModel>, action: SelectClassification): Observable<Classification> {
```

Comment:
Should we maybe declare, that there is the possibility of a null response? Or do you think this is not necessary?

Reply:
I honestly don't know.

Reply:
done